dirty_cat/datetime_encoder.py

```diff
@@ -20,9 +20,9 @@ class DatetimeEncoder(TransformerMixin, BaseEstimator):
     Parameters
     ----------
     extract_until : {"year", "month", "day", "hour", "minute", "second", "millisecond", "microsecond", "nanosecond"}, default="hour"
-        Extract up to this granularity, and gather the rest into the "other" feature.
-        For instance, if you specify "day", only "year", "month", "day" and "other" features will be created.
-        The "other" feature will be a numerical value expressed in the "extract_until" unit.
+        Extract up to this granularity. If all features have not been extracted, add the "full" feature, which contains
```

Comment:
Maybe call it "total_time" rather than "full". I had not understood what "full" meant.